vnext/Shared/Shared.vcxitems

```diff
@@ -56,7 +56,16 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)Modules\StatusBarManagerModule.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)Modules\WebSocketModule.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)Networking\OriginPolicyHttpFilter.cpp" />
-    <ClCompile Include="$(MSBuildThisFileDirectory)Networking\WinRTHttpResource.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)Networking\RedirectHttpFilter.cpp">
+      <!-- Forced pch.h prevents unsetting WINRT_LEAN_AND_MEAN -->
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
```

Comment:
>ecompiledHeader>NotUsing</Precompil [](http://example.com/codeflow?start=9&length=35)

Note this will SLOW down compliation as you are increasing the cricitical path.

MsBuild is single threaded inside a project. And therefore it will first compile all files with same arguments togther (in parralel) but then After that that will run CL for these two files as they will have different commandline arguments.

Reply:
Noted. We could try to make this exclusive to UWP+FastBuild.
For that scenario, this change is necessary because the PCH forces `WINRT_LEAN_AND_MEAN`.

Another option is to update or copy the PCH to prevent this.

Reply:
Found a better approach.
Remove header <winrt/Windows.Web.Http.Filters.h> from the PCH.
It is not broadly used (only by on other source file), so its performance gain was likely negligible.